packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js

```diff
@@ -147,7 +142,9 @@ TableRow.propTypes = {
 };
 
 TableRow.defaultProps = {
+  enableBulkActions: true,
   onDelete: () => {},
+  value: false,
```

Comment:
There was two `TableRow.defaultProps` defintions. I merged in one either the original code is one replacing the other... isn't?.

Let me know if I should just remove the first definition.